README.md

```diff
@@ -14,13 +14,15 @@ The buildpack will do the following:
 * Contributes `rustup-init` to a layer marked `cache` with command on `$PATH`
 * Executes `rustup-init` with the output written to a layer marked `build` and `cache` with installed commands on `$PATH`
 * Executes `rustup` to install a Rust toolchain to a layer marked `build` and `cache` with installed commands on `$PATH`
+* If `$BP_RUST_TARGET` is set, executes `rustup target add` to install an additional Rust target.
 
 ## Configuration
 | Environment Variable      | Description                                                                                                                                                                                                                                                                  |
 | ------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `$BP_RUSTUP_ENABLED`      | Configure rustup to be enabled. This means that rustup will be used to install Rust. Default value is `true`. Set to false to use another Rust toolchain provider like [rust-dist](https://github.com/paketo-community/rust-dist).                                           |
 | `$BP_RUST_TOOLCHAIN`      | Rust toolchain to install. Default `stable`. Other common values: `beta`, `nightly` or a specific versin number. Any [acceptable value for a toolchain](https://dev-doc.rust-lang.org/beta/edition-guide/rust-2018/rustup-for-managing-rust-versions.html) can be used here. |
 | `$BP_RUST_PROFILE`        | Rust profile to install. Default `minimum`. Other acceptable values: `default`, `complete`. See [Rustup docs for profile](https://rust-lang.github.io/rustup/concepts/profiles.html).                                                                                        |
+| `$BP_RUST_TARGET`         | Additional Rust target to install. Default ``, so nothing additional is installed. Run `rustup target list` to see what valid targets exist.                                                                                                                                 |
```

Comment:
Maybe mention that if the `tiny` stack is used, rustup will automatically add the compilation target for `musl` and use the `rustup` musl version?

And would also make sure, that both can be overridden in case this is not the wanted behavior for whatever reason.

Reply:
>Maybe mention that if the tiny stack is used, rustup will automatically add the compilation target for musl and use the rustup musl version?

Good feedback. Done.

>And would also make sure, that both can be overridden in case this is not the wanted behavior for whatever reason.

Yes, that's the current implementation. If the user populates this value, then the buildpack does nothing. User always has the final say.